views/github-create-branch.hbs

```diff
@@ -61,8 +61,7 @@
         </a>
         <aui-inline-dialog id="show-no-repo-popup" alignment="bottom left" responds-to="hover">
           <div>You need to configure your GitHub repositories to your Jira site to see them here.</div>
-          <!--TODO: Add URL after confirmation-->
-          <a href="#" class="configure-link">Configure GitHub Integration</a>
+          <a href="{{jiraHost}}/plugins/servlet/ac/com.github.integration.production/github-post-install-page" target="_blank" class="configure-link">Configure GitHub Integration</a>
```

Comment:
This page is either shown inside the iframe or in a separate new tab. If it was inside iframe, we needed to use `AP.navigator` and if its new tab, will have to provide URL. 
So, just hard-coded the prod URL instead.

Reply:
Nit: Integration should be lowercased.

<img width="346" alt="Screen Shot 2022-10-17 at 3 48 17 pm" src="https://user-images.githubusercontent.com/37155488/196091648-88035203-e82a-4d15-8646-7395baf0ec4d.png">

Reply:
Casing corrected